splunk_add_on_ucc_framework/global_config_update.py

```diff
@@ -123,17 +123,27 @@ def _handle_xml_dashboard_update(global_config: global_config_lib.GlobalConfig)
 
 def handle_global_config_update(global_config: global_config_lib.GlobalConfig) -> None:
     """Handle changes in globalConfig file."""
-    current_schema_version = global_config.schema_version
-    version = current_schema_version if current_schema_version else "0.0.0"
+    current_schema_version = global_config.schema_version or "0.0.0"
+
     logger.info(f"Current globalConfig schema version is {current_schema_version}")
 
+    if _version_tuple(global_config.meta.get("_uccVersion", "0.0.0")) >= _version_tuple(
+        "5.52.0"
+    ):
```

Comment:
`_uccVersion` is a build time attribute, will it be `None` here always?

Let's revert to the previous code and let's merge this one.

Reply:
It would be `None` for the add-ons migrated from AOB to UCC and would always be present for UCC add-ons.. This issue was found from the AOB to UCC migration smoke test.

Reply:
Oh, I see.

Can you please comment this in the code itself? I would forget about it in a week time  😄